example/spec/parallel/ingest/ingestFromPdrWithChildWorkflowMetaSpec.js

```diff
@@ -124,7 +124,7 @@ describe('The DiscoverAndQueuePdrsChildWorkflowMeta workflow', () => {
         config.stackName,
         config.bucket,
         workflowName,
-        { name: addedCollection.name, version: addedCollection.version },
+        { name: addedCollection[0].name, version: addedCollection[0].version },
```

Comment:
Wow. How was this even working before?

Reply:
The test itself doesn't actually check the final state of the child workflow, just that it *did* result in one -- and since we're not evaluating the final database state (for many of our integration tests like this one) there's no failure other than a DLQ entry/log/etc.       There's nothing stopping this test from firing off a workflow with a garbage object for a collection.

Reply:
😢